rancher/resource_rancher_environment.go

```diff
@@ -131,7 +131,7 @@ func resourceRancherEnvironmentCreate(d *schema.ResourceData, meta interface{})
 	d.SetId(newEnv.Id)
 	log.Printf("[INFO] Environment ID: %s", d.Id())
 
-	return resourceRancherEnvironmentUpdate(d, meta)
+	return resourceRancherEnvironmentCreateOrUpdateProjectMembers(d, meta)
```

Comment:
That name though… Maybe `resourceRancherEnvironmentReplaceMembers`?

Reply:
`resourceProjectMembersCreateOrUpdate` maybe? I am a little bit reluctant to ditch createOrUpdate part because method is used on both.

Reply:
Might as well remove `resource` as well then, and make it `projectMembersCreateOrUpdate`